### ISSUE TITLE
Document the between TO filter operator 

### ISSUE THREAD

**Issue by Kerollmops** [meilisearch/documentation#1110]
It would be great to document the between i.e. TO filter operator in [the filters expression page](https://docs.meilisearch.com/reference/features/filtering_and_faceted_search.html#filter-expressions)

```
price 32 TO 45.5
NOT price 20 TO 300
```

is equivalent to

```
price >= 32 AND price <= 45.5
NOT (price >= 20 AND price <= 300)
```

**Comment by maryamsulemani97** [comment]
@Kerollmops 
Can you please share any additional information that needs to be added to the documentation?

**Comment by maryamsulemani97** [comment]
resolved in PR #1221
